### PR TITLE
[Tech Debt] Remove Old Dead Code (pre CIP-20)

### PR DESCRIPTION
**Pull request by bh2smith**
The payout script no longer needs to hold onto any code related Pre CIP 20, since it exists in a tagged branch. The only reason we held onto it originally was for the transition period. This PR removes a TON of dead code.

# Test Plan

Existing CI + A dry run of the payouts to validate that nothing was deleted that should still be here).

I ran the payouts for the accounting period July 1 to 7  with the command

```
python -m src.fetch.transfer_file --dry-run True
```
and generated this CSV
[transfers-2023-07-01-to-2023-07-08.csv](https://github.com/cowprotocol/solver-rewards/files/11992410/transfers-2023-07-01-to-2023-07-08.csv)

having these normal totals

```
Performance Reward: 183346.9394
Participation Reward: 199960.0606

Total ETH Funds needed: 44.4472
Total COW Funds needed: 382848.7395
```

Which can be compared with [the dashboard](https://dune.com/cowprotocol/cow-solver-rewards?StartTime=2023-07-01+00%3A00%3A00&EndTime=2023-07-08+00%3A00%3A00&EndTime_d3bba3=2023-07-08+00%3A00%3A00&StartTime_d3f009=2023-07-01+00%3A00%3A00)

Note that this PR does not resolve the [coinpaprika e2e issue](https://github.com/cowprotocol/solver-rewards/issues/308).

### CODE REVIEW

File: queries/orderbook/order_rewards.sql
Comment (gentrexha) [inline queries/orderbook/order_rewards.sql:1]: Why are we getting rid of these queries?

File: src/utils/script_args.py [hunk 1]
```diff
@@ -64,6 +56,7 @@ def generic_script_init(description: str) -> ScriptArgs:
         "Primarily intended for deployment in staging environment.",
         default=False,
     )
+    # TODO: this should be per token (like list[tuple[Token,minAmount]])
```
Comment (gentrexha) [inline src/utils/script_args.py:59]: I would reference to a ticket for this. What do you think?

**Review by gentrexha — APPROVED** [review]
Left two discussion comments. Other than that, LGTM!

File: queries/orderbook/order_rewards.sql
Comment (bh2smith) [reply queries/orderbook/order_rewards.sql:1]: We no longer reward orders. This query is no longer touched by the code. We reward batches now.

File: src/utils/script_args.py (hunk 1 above)
Comment (bh2smith) [reply src/utils/script_args.py:59]: Good thinking. Didn't want to change it here because it also requires changing deployment parameters.

Comment (bh2smith) [reply src/utils/script_args.py:59]: Issue here: https://github.com/cowprotocol/solver-rewards/issues/314

**Pull request merged**